docs/autostart_systemd.rst

```diff
@@ -89,7 +89,13 @@ type the following command in the terminal, still by adding the instance name af
 .. warning:: If the service doesn't stop in the next 10 seconds, the process is killed.
     Check your logs to know the cause of the error that prevents the shutdown.
 
-To view Red’s log, you can acccess through journalctl:
+To disable the service, run the following, adding the instance name after the **@**:
```

Comment:
To not give a wrong impression, this should say that what the command does is making the bot not start on boot anymore:
```suggestion
To set the bot to not start on boot anymore, you must disable the service by running the following command, adding the instance name after the **@**:
```